### ISSUE TITLE
[Question]: Local LLM API based on Vllm is not usable？

### ISSUE THREAD

**Issue by Bkk-shuai** [infiniflow/ragflow#3144]
### Describe your problem

my local API is similar in format to Openai like http://xxx.xx.xx.xxx:5000/v1 and has a api-key  how can I use this API in RAGFlow?

**Comment by Bkk-shuai** [comment]
我在Vllm启动项删除了 API-KEY后 可以正常导入， 但是加入api-key后 就提示401

**Comment by KevinHuSh** [comment]
Did you use 'OpenAI-API-Compatible' or something else to connect your vllm server?

**Comment by Bkk-shuai** [comment]
I used OpenAI API Compatible, but the current situation is that the locally deployed LLM needs to remove the API key to import normally

**Comment by KevinHuSh** [comment]
401 means unautherized. I'm not sure if your API key is correct.

**Comment by Bkk-shuai** [comment]
yes i know that but when I use APIfox to use my local LLM with this API key the response is 200
